ledger-service/http-json-perf/src/main/scala/com/daml/http/perf/scenario/AsyncQueryConstantAcs.scala

```diff
@@ -49,12 +49,13 @@ class AsyncQueryConstantAcs
       pause(1.second)
     }
     .exec(
-      1.to(numberOfRuns).map(runId => exec(query(runId)))
+      1.to(numberOfRuns / defaultNumUsers).map(runId => exec(query(runId)))
     )
 
   setUp(
-    fillAcsScenario(wantedAcsSize, silent = true).inject(atOnceUsers(1)),
-    asyncQueryScenario.inject(atOnceUsers(1)),
+    fillAcsScenario(wantedAcsSize, silent = true).inject(atOnceUsers(defaultNumUsers)).andThen {
```

Comment:
makes scenario runs sequential, just cleaner since second query scenario needs to wait for the ACS to be written anyway.